src/index.ts

```diff
@@ -153,10 +157,13 @@ export class Paperless {
   }
 
   /**
-   * Lists all tags
+   * Lists all or filtered tags
+   * @param filter - Filter for the result
    */
-  public async getTags(): Promise<TagParsed> {
-    const response = await this.instance.get<TagResult>(`/tags/`);
+   public async getTags(filter: TagFilter): Promise<TagParsed> {
```

Comment:
Same as above, make the filters optional so it won't break current implementations
```suggestion
   public async getTags(filter?: TagFilter): Promise<TagParsed> {
```